docs/installation.md

```diff
@@ -89,9 +158,19 @@ Sensors used for plant phenotyping can be carried on vectors such as UAVs or fie
 The `Vectors` menu displays the list of vectors that have been declared by the user on Phis.
 The vector list is preceded by a research bar enabling the user to filter registered vectors by their Alias, type, brand and optionnal attributes, such as the date of their purchase, the date of their first use or their serial number.
 
-
 ![vector-menu](img/vector-list.png)
 
+From **Home / Vectors**, one can click on the eye icon on the right of any listed vector to see its information sheet.
+
+<!---
+![vector-information](img/vector-information.png)
```

Comment:
no screenshot has been made yet of the information sheet of a vector since depending on the version of PHIS i'm working on, either "update and "add documents" buttons are displayed, either only "update" is displayed (no "add annotation" button)